service/src/test/java/bio/terra/pipelines/dependencies/cbas/CbasClientTest.java

```diff
@@ -0,0 +1,42 @@
+package bio.terra.pipelines.dependencies.cbas;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import bio.terra.cbas.api.MethodsApi;
+import bio.terra.cbas.api.PublicApi;
+import bio.terra.cbas.api.RunSetsApi;
+import bio.terra.cbas.api.RunsApi;
+import bio.terra.pipelines.testutils.BaseContainerTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+class CbasClientTest extends BaseContainerTest {
+  @Autowired CbasClient cbasClient;
+  String cbasBaseUri = "cbasBaseUri";
+  String authToken = "authToken";
+
+  @Test
+  void TestWdsClientApis() {
+
+    PublicApi publicApi = cbasClient.publicApi(cbasBaseUri, authToken);
+
+    assertEquals(cbasBaseUri, publicApi.getApiClient().getBasePath());
+    assertTrue(publicApi.getApiClient().isDebugging());
```

Comment:
what sets this to True? is that the default that the CBAS client sets? I guess what I'm really asking is, why are we testing this?

Reply:
its set to true in the application-test.yml file